src/scripts/run_backup.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+S3_BUCKET=$1
+if [ -z "${S3_BUCKET}" ]; then
+    echo "Missing argument: S3 Bucket"
+    exit 1
+fi
+
+S3_PATH=$2
+if [ -z "${S3_PATH}" ]; then
+    echo "Missing argument: S3 Path"
+    exit 1
+fi
+
+S3_ACCESS_KEY=$3
```

Comment:
I think we can pass credentials as env variables (not as cmd arguments). It is a typical security concern.
https://smallstep.com/blog/command-line-secrets/
https://github.com/percona/percona-xtrabackup/blob/8.0/storage/innobase/xtrabackup/src/xbcloud/xbcloud.cc#L316-L320

in any case, can be handled as a separate ticket/PR.

Reply:
addressed in [2f28871](https://github.com/canonical/mysql-k8s-operator/pull/136/commits/2f288712cb675a55668e3c75896aaba6feebe24d). passing in arguments as env vars, and also running the commands with the `mysql` POSIX user in the mysql container

Reply:
not fixed yet, credentials are still command-line arguments 😞 anyone still can get them via the `ps` command.
credential should be environment variables only, without passing via arguments.

Reply:
actually use env vars for `ACCESS_KEY_ID` and `SECRET_ACCESS_KEY` in aedeff6b9921c57431722fcf5a827a4509226ae3. also, use stdin for `MYSQL_PASSWORD`